src/main/java/com/griefcraft/model/Protection.java

```diff
@@ -1022,6 +1012,49 @@ public String toString() {
                         creation, flagStr, lastAccessed);
     }
 
+    /**
+     * New protection information format
+     *
+     * @param sender Command Sender
+     * @see Protection#toString()
+     * @since 2.2.2
+     */
+    public void sendProtectionInfo(CommandSender sender) {
+        LWC lwc = LWC.getInstance();
+
+        // format the flags prettily
+        StringBuilder flagStr = new StringBuilder();
+
+        for (Flag.Type type : Flag.Type.values()) {
+            ChatColor color = ChatColor.RED;
+            for (Flag flag : flags.values()) {
+                if (type.equals(flag.getType())) {
+                    color = ChatColor.GREEN;
+                    break;
+                }
+            }
+            flagStr.append(color).append(type.toString().toLowerCase()).append(" ");
+        }
+
+        if (flagStr.toString().endsWith(" ")) {
+            flagStr = new StringBuilder(flagStr.substring(0, flagStr.length() - 1));
+        }
+
+        // format the last accessed time
+        String lastAccessed = TimeUtil
+                .timeToString((System.currentTimeMillis() / 1000L)
+                        - this.lastAccessed);
+
+        /*
+        TODO: Translate
+        if (!lastAccessed.equals("Not yet known")) {}
+        */
+
+        BlockCache blockCache = BlockCache.getInstance();
+        Material blockType = blockCache.getBlockType(blockId);
```

Comment:
You get the block type here but don't use it. Looks like you might have forgotten it, since old protection info does display it.